ramls/examples/providers/providers_providerId_packages_get_400_response.json

```diff
@@ -0,0 +1,10 @@
+{
+  "errors": [
+    {
+      "title": "Parameter OrderBy should be by PackageName when parameter Search is empty."
```

Comment:
error message "Parameter OrderBy should be by PackageName when parameter Search is empty" is an error returned from RM API, mod-kb-ebsco-java doesn't have parameter OrderBy, in other endpoints we return "Search parameter cannot be empty" for this case

Reply:
Thanks, updated